doc/src/contribute/observability.md

```diff
@@ -210,26 +210,39 @@ To see nested spans visualized with [Jaeger](https://www.jaegertracing.io), do t
 
 ### Memory profiling
 
-jemalloc has a memory profiling mode that can be enabled at runtime with environment variables. See:
+Sui uses the jemalloc memory allocator by default on most platforms, and there is code that enables automatic
```

Comment:
Should we mention using the docker image?

Reply:
Yes this is mentioned in the next paragraph